auslib/blobs/settings.py

```diff
@@ -0,0 +1,43 @@
+import re
+from auslib.blobs.base import Blob
+
+
+INDENT = 4 * ' '
+SETTING_TMPL = ('<setting id="%(id)s" '
+                'lastModified="%(last_modified)s"/>')
+
+
+class SettingsBlob(Blob):
+    format_ = {
+        'name': None,
+        'schema_version': None,
+        'settings': {
+            '*': {
+                'version': None,
+                'last_modified': None
+            }
+        }
+    }
+
+    def __init__(self, **kwargs):
+        Blob.__init__(self, **kwargs)
+        if "schema_version" not in self:
+            self["schema_version"] = 2000
+
+    def shouldServeUpdate(self, updateQuery):
+        return False
```

Comment:
I don't think you want False here...it means any update requests that point at Settings blobs will always get nothing back. I'm pretty sure you want to always return True here, since the client is going to be smart enough to only ping Kinto when the last_modified has changed.